examples/java/src/main/java/org/apache/beam/examples/complete/game/LeaderBoard.java

```diff
@@ -204,8 +213,8 @@ public static void main(String[] args) throws Exception {
         pipeline
             .apply(
                 PubsubIO.readStrings()
-                    .withTimestampAttribute(GameConstants.TIMESTAMP_ATTRIBUTE)
-                    .fromTopic(options.getTopic()))
+                    .fromSubscription(options.getSubscription())
```

Comment:
Same here, it should be conditional on whether `options.getSubscription()` is provided by the user, and otherwise use a topic.